backend/tests/conftest.py

```diff
@@ -1537,6 +1539,23 @@ def mock_failing_gql(
         "the chain was reorganized while executing the query"
     )
 
+    # Mock datetime.datetime.now to return a fixed time
```

Comment:
i think the code is self-explanatory and we can remove all these redundant comments